lib/generators/komponent/install_generator.rb

```diff
@@ -18,18 +22,29 @@ def modify_webpacker_configuration
       end
 
       def move_webpacker_default_structure
+        return unless File.directory?(webpacker_default_structure)
+
         run("mv #{webpacker_default_structure}/* #{komponent_root_directory}")
       end
 
       def create_komponent_default_structure
+        return if File.exist?(components_directory.join("index.js"))
+
         empty_directory(components_directory)
         create_file(components_directory.join("index.js"))
       end
 
       def create_stimulus_file
+        return if File.exist?(stimulus_application_path)
+
         create_file(stimulus_application_path, stimulus? ? stimulus_application_template : "")
       end
 
+      def append_to_application_configuration
+        application "config.autoload_paths += Dir[config.root.join('#{relative_path_from_rails}/components')]"
```

Comment:
Why

`config.autoload_paths += Dir[config.root.join('#{relative_path_from_rails}/components')]`

instead of simply:

`config.autoload_paths << Rails.root.join('#{relative_path_from_rails}/components')`

?